### DISCUSSION TITLE
Should mirror_server and mirror_server_stop be installed?

### DISCUSSION THREAD

**Discussion by opoplawski** [HDFGroup/hdf5#4933] [General]
I'm looking to switch the Fedora hdf5 package build to cmake.  It install `mirror_server` and `mirror_server_stop` into /usr/bin.  Is that really wanted?  Pretty generic name if so.

**Comment by opoplawski** [comment]
Also, this is installed: `/usr/include/H5fortran_types.F90` which seems very odd.

**Reply by brtnfld** [reply]
There should be no reason or need to install the Fortran file. So yes, that is odd.

**Comment by jhendersonHDF** [comment]
@opoplawski I can't speak to the Fortran file, but, in retrospect, I think that yes the mirror server and mirror server stop executables should have probably at least had an "h5" prefix. That said, there's also the question of whether these should even be installed when the mirror VFD isn't built. CMake appears to always build and install them by default, while Autotools only does when the optional mirror vfd is being built. I'll bring this up for discussion.

**Comment by opoplawski** [comment]
Thank you for the responses.  I've filed #4984  and #4985
